### PR TITLE
[QA] Missing requirement

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers 
@geonexus 

#### Description
Missing requirement for acceptance tests

### CODE REVIEW

File: tests/acceptance/requirements.txt [hunk 1]
```diff
@@ -6,3 +6,4 @@ oslo.utils==1.7.0
 PyHamcrest>=1.8.3
 git+https://github.com/qaenablers/python-qautils.git@v1.2.0
 pika==0.10.0
+behave==1.2.5
```
Comment (flopezag) [inline tests/acceptance/requirements.txt]: It should not be part of the requirements_dev.txt?

**Comment by jframos** [comment]
For the moment, we are considering the acceptance test project as an independent 'subproject' in the repo of the component. This subproject manages its own requirement.txt, installation procedure and documentation.

File: tests/acceptance/requirements.txt (hunk 1 above)
Comment (jframos) [inline tests/acceptance/requirements.txt]: For the moment, we are considering the acceptance test project as an independent 'subproject' in the repo of the component. This subproject manages its own requirement.txt, installation procedure and documentation.

Comment (flopezag) [inline tests/acceptance/requirements.txt]: NTC

**Pull request merged**